src/frontend/src/views/Upload/components/Metadata/index.tsx

```diff
@@ -92,9 +92,7 @@ export default function Metadata({
             `Check local requirements for reporting to public health authorities. These may not be met by uploading samples to Aspen.`,
             <p key="1">
               Please read our{" "}
-              <NewTabLink href={ROUTES.PRIVACY}>
-                Privacy Policy
-              </NewTabLink>{" "}
```

Comment:
The space got lost here. I'm never totally sure when spaces get auto-inserted or lost, but it looks like without the forced one here the spacing gets lost when rendered:
![image](https://user-images.githubusercontent.com/89553795/135921948-0cdc7580-86c8-4abc-87db-f63bf13ad2ab.png)